### PR TITLE
ConfirmDialog component tests

### PR DESCRIPTION
**Pull request by andresmgot**
Fixes: #347 

Added tests for ConfirmDialog.

Removed items that were not used.

### CODE REVIEW

File: dashboard/src/components/ConfirmDialog/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { mount } from "enzyme";
+import * as React from "react";
+import * as Modal from "react-modal";
+
+import ConfirmDialog from "./index";
+
+it("renders loading message", () => {
+  const wrapper = mount(
```
Comment (prydonius) [inline dashboard/src/components/ConfirmDialog/index.test.tsx]: I don't think you need to use `mount` here. Please use `shallow` where possible. See difference here: https://gist.github.com/fokusferit/e4558d384e4e9cab95d04e5f35d4f913

File: dashboard/src/components/ConfirmDialog/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      modalIsOpen={true}
+      loading={true}
+      onConfirm={jest.fn()}
+      closeModal={jest.fn()}
+    />,
+  );
+  expect(wrapper.find(Modal).text()).toContain("Loading ...");
+});
+
+it("perform action on confirm", () => {
```
Comment (prydonius) [inline dashboard/src/components/ConfirmDialog/index.test.tsx]: s/perform/performs

File: dashboard/src/components/ConfirmDialog/index.tsx [hunk 3]
```diff
@@ -33,22 +31,21 @@ class ConfirmDialog extends React.Component<IConfirmDialogProps, IConfirmDialogS
...
-          {this.state.error && (
-            <div className="padding-big margin-b-big bg-action">{this.state.error}</div>
-          )}
           {this.props.loading === true ? (
             <div> Loading ... </div>
           ) : (
             <div>
               <div> Are you sure you want to delete this? </div>
-              <button className="button" onClick={this.props.closeModal}>
+              <button id="cancel" className="button" onClick={this.props.closeModal}>
```
Comment (prydonius) [inline dashboard/src/components/ConfirmDialog/index.tsx]: A better option is to use React's `ref` - https://reactjs.org/docs/refs-and-the-dom.html https://airbnb.io/enzyme/docs/api/ReactWrapper/ref.html.

**Comment by prydonius** [comment]
@andresmgot do you plan on picking this one back up?

**Comment by andresmgot** [comment]
Let's close this for the moment

**Pull request closed without merging**